src/compose/ComposeMenu.js

```diff
@@ -183,17 +184,20 @@ class ComposeMenu extends PureComponent<Props> {
               />
             )}
             <IconImage
+              accessibilityLabel="Gallery"
               style={this.styles.composeMenuButton}
               size={24}
               onPress={this.handleImagePicker}
             />
             <IconCamera
+              accessibilityLabel="Camera"
```

Comment:
Maybe, "Send an image from the camera"?

Reply:
Thank you @chrisbobbe for your precise reply. I will consider the points mentioned by you and do the following changes.